### ISSUE TITLE
Strange invisible vertical slab glitch leading to a crash

### ISSUE THREAD

**Issue by Korumaku** [CJMinecraft01/DoubleSlabs#263]
Somehow created an invisible vertical slab block when breaking two vertical cobblestone slabs in one blockspace. Its name showing up through the Jade mod had changed from what it originally was too. This glitched block was unbreakable in survival mode, so I switched to creative. Breaking this glitched vertical slab in creative immediately crashed my game. Log attached.

[crash-2023-01-26_16.37.51-client.txt](https://github.com/CJMinecraft01/DoubleSlabs/files/10513996/crash-2023-01-26_16.37.51-client.txt)

**Comment by underpigeon** [comment]
other than the crash is breaking the slab in creative a viable fix to it?

**Comment by Korumaku** [comment]
Yep, although it crashed, it did get rid of the slab upon reopening the game. So thankfully it doesn’t damage the save.

**Comment by ianwilliamson5** [comment]
any two vert cobble slabs produce this effect please fix

**Comment by Menzicat** [comment]
My crash report looks a little different from OP's, but they both seem to stem from the same null pointer
`java.lang.NullPointerException: Cannot invoke "net.minecraft.world.level.block.state.BlockState.getSoundType(net.minecraft.world.level.LevelReader, net.minecraft.core.BlockPos, net.minecraft.world.entity.Entity)" because the return value of "cjminecraft.doubleslabs.api.IBlockInfo.getBlockState()" is null`

The invisible ghost slab is still in place and I can't get rid of it. Any ideas how to do so?
